test/integration/narrow/payment-requests.test.js

```diff
@@ -0,0 +1,122 @@
+const db = require('../../../app/data')
+jest.mock('ffc-messaging')
+let createServer
+let server
+let paymentRequest
+
+describe('schemes routes', () => {
+  beforeEach(async () => {
+    await db.sequelize.truncate({ cascade: true })
+
+    await db.scheme.create({
+      schemeId: 1,
+      name: 'SFI'
+    })
+
+    paymentRequest = {
+      completedPaymentRequest: 1,
+      paymentRequestId: 1,
+      schemeId: 1,
+      invoiceNumber: 'InvoiceNumber'
```

Comment:
Do we have a defined rule set for what invoice numbers look like and can we think (in another ticket) to incorporate that into our schema validation checks and test cases?

Reply:
ish, we know they'll be strings and we know what they can be like for SFI Pilot.  Elsewhere in the service we've avoided being too restrictive on invoice formats because we don't know what we're going to get in the coming months for Vets, SFI and Lump Sums.